hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2819,6 +2819,15 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.enable.key.path.lock</name>
+    <tag>OZONE, OM</tag>
+    <value>false</value>
+    <description>If true, the fine-grained KEY_PATH_LOCK functionality is enabled.
+      If false, it is disabled.
```

Comment:
You can say, 

Defaults to false.

Reply:
Thank you, I have incorporated the change!